SemiconductorTestLibrary.Extensions/source/InstrumentAbstraction/DCPower/Types.cs

```diff
@@ -150,6 +150,14 @@ public class DCPowerWaveformAcquisitionSettings
         /// The measure trigger type.
         /// </summary>
         public DCPowerMeasureTriggerType MeasureTriggerType { get; set; }
+
+        /// <summary>
+        /// Creates an object to define and set DCPower waveform acquisition specific settings.
+        /// This object is used as a parameter for configuring waveform acquisition settings for a DCPower channel. <see cref="Measure.ConfigureAndStartWaveformAcquisition(DCPowerSessionsBundle, double, double)"/>
```

Comment:
Now that I checked its usage in the method you referenced, I think it would be more appropriate to say "This object is used to store waveform acquisition settings of a DCPower channel. <see...".

Sorry for the back and forth, it should be the last time.

Reply:
Updated accordingly